### DISCUSSION TITLE
Notepad++ reports update but does not install

### DISCUSSION THREAD

**Discussion by 411621** [chocolatey-community/chocolatey-packages#2280] [General]
Notepad++ reports update but does not install

I have Notepad++ 8.5.5 installed via chocolatey.
There are 2 entries: **Notepad++** and **Notepad++ (installed)**.
Both show as version 8.5.5

Under the entry "**Notepad++ (installed)**" it shows that there is an update available "**8.5.6 out of date**"
However, whenever I attempt to update it it fails.
I tried _purging outdated_ and use chocolatey cleaner app but all to no avail.

The console output has some messages but it scrolls by quickly and I haven't figured out how to stop it or output to file so I can read it.
[Attached is a screen clip showing what happens](https://www.mediafire.com/file/6xg4ausc7kt5tmz/20230820_095753.mp4/file) when trying to update aforementioned.

**Comment by Starmania** [comment]
A new launch of a test run by admin [here](https://community.chocolatey.org/packages/notepadplusplus/8.5.6#description)  will solve the problem. This is due to notepad plus plus.install not available at the moment of the run, but he finished...

**Reply by Starmania** [reply]
So just wait that a moderator or the maintainer restart the verification process...

**Comment by TheCakeIsNaOH** [comment]
Retry it now, as `notepadplusplus` v8.5.6 is now approved.
